cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -51,6 +54,7 @@ const (
 	migAutoDiscovererKeyPrefix   = "namePrefix"
 	migAutoDiscovererKeyMinNodes = "min"
 	migAutoDiscovererKeyMaxNodes = "max"
+	concurrentGceRefreshes       = 10
```

Comment:
This really needs to a configurable via an flag with a default of 1. This can affect project quota in major ways.